CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 
 ### Bug fixes
 
+* [#315](https://github.com/rubocop-hq/rubocop-rails/pull/315): Allow to use frozen scope for `Rails/UniqueValidationWithoutIndex`. ([@krim][])
+
```

Comment:
Can you remove the redundant blank line and squash your commits into one?

Reply:
Ok, I did it.